### DISCUSSION TITLE
Add CC BY-NC licence to quarto book

### DISCUSSION THREAD

**Discussion by franzeder** [quarto-dev/quarto-cli#4792] [Q&A]
How can I add a licence (e.g., CC BY-NC) to a quarto book?

I would like to print this licence (maybe including the respective figures) at my Quarto Book Website ([https://www.politik-wissenschaft.at/](https://www.politik-wissenschaft.at/)) next to the DOI.

How can I do that?

**Comment by dragonstyle** [comment]
Right now, I can't think of a way to do this short of replacing the title-block template with your own title block template. 

More here: https://quarto.org/docs/journals/templates.html#html-partials

That would be a nice addition to books (as would some of the features of citable articles). The general citable article support is tracked here: https://github.com/quarto-dev/quarto-cli/issues/3838. I will open a new issue to add optionally add license to the title block as well.
